PocketKit/Sources/Analytics/AppEvents/Reader.swift

```diff
@@ -76,7 +76,17 @@ public extension Events.Reader {
             contentEntity: ContentEntity(url: url),
             uiEntity: UiEntity(
                 .button,
-                identifier: "reader.view_original"
+                identifier: "reader.view-original"
+            )
+        )
+    }
+
+    static func openExternalLink(url: URL) -> ContentOpen {
```

Comment:
@dskuza can you define an external link open in the description. im unclear if that means opening in web view? or said differently... how does one open an external link in reader?

Reply:
yeah, i'll go back and leave a comment, but…

an external link is a link that exists within the reader view (not a webview), usually in a text component. when a link is tapped in the reader, it is an "external link". alternatively, if you long-press a link, there is an "open" action that is also the same thing.